jazz_scm-webhook/components/config.js

```diff
@@ -22,20 +23,10 @@
 **/
 
 var getStageConfig = (event) => {
-  var stage;
-  
-  if (event && event.awslogs && event.awslogs.data) {
-      // cw events default to dev
-      stage = 'dev';
-  }else {
-      stage = event.stage
-  } 
-  
-  var configObj;
-  
-  if (stage) {
-      configObj = require(`../config/${stage}-config.json`);
-  } 
+  var stage = event.stage,

```

Comment:
I had condensed this code to [this](https://github.com/tmobile/jazz/blob/master/jazz_usermanagement/components/config.js#L23). If we are changing this for performance improvements, the one [here](https://github.com/tmobile/jazz/blob/master/jazz_usermanagement/components/config.js) is perhaps more optimized. @svidhani: What do you think?

Reply:
even better :). thanks